src/add-to-dagnode-transform.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const async = require('async')
+const getDagNode = require('./get-dagnode')
+
+// transform { Hash: '...' } objects into { path: 'string', node: DAGNode }
+module.exports = function (err, res, send, done) {
+  if (err) {
+    return done(err)
+  }
+  async.map(res, function map (entry, next) {
```

Comment:
roger. Due to time constraints, can we make that a issue to hit later? It would be good to explain in one paragraph (or even one line -> "async is a huge dependency, makes our browser lib fat") so that @noffle and other contribs try to avoid it in the future?